### PR TITLE
Login Layout

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Add login layout.

#### Galaxy S (or similar)
Screenshot_1622134177

#### Pixel 2/3 (or similar)
Screenshot_1622134781

#### Pixel 3XL (or similar)
Screenshot_1622134783

#### iPhone SE (or similar)
simulator_screenshot_34F04289-C383-46B3-BFB5-18C4C713815D

#### iPhone 6/7/8 (or similar)
simulator_screenshot_7E7D1AC4-828B-4EAE-A4D4-7AAF72F1BDC9

#### iPhone 11/12 Base/Pro/Pro Max (or similar)
simulator_screenshot_13D13DCF-FFE0-4D25-BF52-50E350452EE6

## Trello Card

https://trello.com/c/3nos3UZ1/24-n-101-login-maquetado

### CODE REVIEW

File: wbooks/src/app/components/AppNavigator/index.tsx [hunk 1]
```diff
@@ -55,6 +56,7 @@ function AppNavigator() {
         screenOptions={({ route }) => ({
           tabBarIcon: ({ focused }) => <TabBar focused={focused} route={route} />
         })}>
+        <Tab.Screen name="Login" component={Login} options={{ tabBarVisible: false }} />
```
Comment (mcolombo87) [inline wbooks/src/app/components/AppNavigator/index.tsx]: Login is not a member of the tab screen.

**Review by mcolombo87 — CHANGES_REQUESTED** [review]

File: wbooks/src/app/components/AppNavigator/index.tsx (hunk 1 above)
Comment (guidoprinc) [reply wbooks/src/app/components/AppNavigator/index.tsx]: as maxi says, login should be another screen on the stack

File: wbooks/src/app/screens/Login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <ImageBackground source={bcInicio} style={styles.container}>
+      <View style={styles.formContainer}>
+        <Text style={styles.formTitle}>Nombre</Text>
+        <TextInput style={styles.formInput} autoCapitalize="words" />
+      </View>
+      <View style={styles.formContainer}>
+        <Text style={styles.formTitle}>Contraseña</Text>
+        <TextInput style={styles.formInput} autoCapitalize="words" />
+      </View>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Login/index.tsx]: extract this view as a component that receives corresponding props

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/components/AppNavigator/index.tsx (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/app/components/AppNavigator/index.tsx]: Ok

File: wbooks/src/app/screens/Login/index.tsx (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Login/index.tsx]: Done!

File: wbooks/src/app/screens/Login/styles.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  formTitle: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginBottom: 7
+  },
+  formInput: {
+    backgroundColor: COLORS.white,
+    borderRadius: 10,
+    height: 45
+  },
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Login/styles.ts]: delete unused styles

File: wbooks/src/app/screens/Login/index.tsx [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import bcInicio from '@assets/General/bc_inicio.png';
+
+import styles from './styles';
+
+const Login = () => {
+  return (
+    <ImageBackground source={bcInicio} style={styles.container}>
+      <Form title="Email" />
+      <Form title="Password" isSecure={true} />
+      <TouchableOpacity style={styles.loginBtn}>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/Login/index.tsx]: touchables should always have an `onPress` prop, even. if you must write a empty function like `() =>{}`

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/Login/index.tsx (hunk 4 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Login/index.tsx]: Ok, the linter doesn't admit empty arrow function, so I put a console.log

File: wbooks/src/app/screens/Login/styles.ts (hunk 3 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/Login/styles.ts]: Done!

**Review by guidoprinc — APPROVED** [review]

**Review by mcolombo87 — APPROVED** [review]

**Pull request merged**